src/app/public/modules/colorpicker/colorpicker.component.html

```diff
@@ -270,30 +272,34 @@
   >
   </span>
 
-  <button *ngIf="showResetButton && themeName === 'default'"
-    class="sky-colorpicker-reset-button"
-    type="button"
-    [attr.aria-label]="'skyux_colorpicker_reset' | skyLibResources"
-    (click)="resetPickerColor()"
-  >
-    <sky-icon
-      icon="times"
-      size="sm"
+  <ng-template [skyThemeIf]="'default'">
```

Comment:
We know the button will always have the same click action, type, and aria-label. The only thing that really changes is its classes and icon. Seems like it would be a better fit to avoid duplicating templates and just use your directives like so...
```
  <button *ngIf="showResetButton"
    [skyThemeClass]="{
      'sky-colorpicker-reset-button': 'default',
      'sky-btn sky-btn-icon-borderless sky-colorpicker-reset-button-modern': 'modern'
    }"
    type="button"
    [attr.aria-label]="'skyux_colorpicker_reset' | skyLibResources"
    (click)="resetPickerColor()"
  >
    <sky-icon *skyThemeIf="'default'"
      icon="times"
      size="sm"
    >
    </sky-icon>
    <sky-icon *skyThemeIf="'modern'"
      icon="trash"
      iconType="skyux"
      size="lg"
    >
    </sky-icon>
  </button>
  ```
  What do you think?